spec/models/comment_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  before(:example) do
+    @user = User.new(name: 'Lilly', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Poland.')
+    @post = Post.new(author: @user, title: 'Hello', text: 'This is my first post')
+  end
+
+  subject { Comment.new(user: @user, post: @post) }
+
+  it 'name should be present' do
+    subject.user = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'title should be present' do
+    subject.post = nil
+    expect(subject).to_not be_valid
+  end
```

Comment:
- [ ] Great job working on this milestone. Your unit tests for the Comment model are failing. This is because you are writing tests for name should be present and title should be present validations which are not present in the comment.rb model file. I think you should rather write unit test for the method found in the Comment model
![image](https://github.com/MMhaidari/blog-app/assets/62849872/1af698e8-2d26-430b-b2b3-65c6093fd5e1)

Reply:
@t-yanick I don't know how did you run the tests but mine is working fine below snap is for more info
![test cases](https://github.com/MMhaidari/blog-app/assets/99756088/2995af11-ccca-46a6-869e-62012602b271)